requirements/edx/base.txt

```diff
@@ -16,7 +16,7 @@ dealer==2.0.4
 defusedxml==0.4.1
 django-babel-underscore==0.5.1
 markey==0.8  # From django-babel-underscore
-django-celery==3.1.16
+django-celery==3.2.1
```

Comment:
They don't seem to be used on prod, they're all empty when I look. No clue what would be migrating them in clean builds.